llmfoundry/data/text_data.py

```diff
@@ -240,6 +240,9 @@ def build_text_dataloader(
             # fwd all kwargs with **stream allows streaming to check args
             streams.append(Stream(**stream))
 
+    import streaming.base.util
```

Comment:
This import can be at the top of the file

Reply:
➕

Reply:
fixed